src/main/resources/application.yml

```diff
@@ -95,7 +95,10 @@ prison:
 policy:
   session:
     non-association:
+      filter: true
       whole-day: true
+    double-booking:
+      filter: false
```

Comment:
is it false? I suppose we are turning it off until a later date! Do we have to put something in for helm?

Reply:
The requirements might change post mvp, config added to allow the UI to change without re-coding the api.  This config can be overridden, defaults for now to the current requirements.